suggested-content/word-2-vec-service.py

```diff
@@ -27,6 +31,14 @@ def documentCheckPhrases():
     response = checkPhrases(grams)
     return jsonify(results = response)
 
+@app.route('/doc', methods=['POST'])
+def similarDocs():
+    doc = request.json["doc"]   
```

Comment:
Oh sorry - I was spacing out!

On 11 November 2015 at 14:01, Lindsey Dew <notifications@github.com> wrote:

> In suggested-content/word-2-vec-service.py
> <https://github.com/guardian/proposer-suggested-content/pull/27#discussion_r44534288>
> :
>
> > @@ -27,6 +31,14 @@ def documentCheckPhrases():
> >      response = checkPhrases(grams)
> >      return jsonify(results = response)
> >
> > +@app.route('/doc', methods=['POST'])
> > +def similarDocs():
> > +    doc = request.json["doc"]
>
> Yeah this is an endpoint to process to a POST request (from composer say)
> which contains a body of text and can be analysed against our documents in
> memory. The method for processing the CAPI file is called loadDocuments
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/proposer-suggested-content/pull/27/files#r44534288>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396